src/dql_resolution.erl

```diff
@@ -8,7 +8,9 @@
 %%%-------------------------------------------------------------------
 -module(dql_resolution).
 
--export([resolve/2, propagate/1, start_time/1]).
+% -export([resolve/2, propagate/1, start_time/1]).
+
+-compile([export_all]).
```

Comment:
I'm not a fan of export all, specific exports serve a reason of declaring a clear interface and making looking up functions easier.